lib/internal/Magento/Framework/View/Layout/Generator/Block.php

```diff
@@ -299,7 +299,7 @@ protected function generateAction($block, $methodName, $actionArguments)
         $profilerKey = 'BLOCK_ACTION:' . $block->getNameInLayout() . '>' . $methodName;
         \Magento\Framework\Profiler::start($profilerKey);
         $args = $this->evaluateArguments($actionArguments);
-        call_user_func_array([$block, $methodName], $args);
+        call_user_func_array([$block, $methodName], array_values($args));
```

Comment:
```suggestion
        call_user_func_array([$block, $methodName], ...$args);
```
@andrewbess @karyna-tsymbal-atwix 
This is looks more clear for me :)

Reply:
Hello @Den4ik 
The proposed changes don't work with associative arrays.
We will change it for `...[$args]`